### DISCUSSION TITLE
How do I create my own GitHub Page

### DISCUSSION THREAD

**Discussion by ecyojaidan** [community/community#22911] [Other Feature Feedback, Questions, & Ideas]
<p>I’m pretty new at GitHub things, and I’m interested in setting up my own GitHub Page (just as a resume site for now). I’m a little unsure of the best way to do this and would love some help! Any advice is very appreciated.</p>

**Comment by winstliu** [comment, marked as answer]
<p>:wave: Hi <a class="mention" href="/ecyojaidan">@ecyojaidan</a>!  In case you weren’t aware, there is a great tutorial on how to get started with GitHub Pages at <a href="https://pages.github.com/" rel="nofollow noopener">https://pages.github.com/</a>.</p>
<p>Basically, if you want your own user page at <em><a href="https://ecyojaidan.github.io" rel="nofollow noopener">https://ecyojaidan.github.io</a></em>, you should first create a repository called “<a href="http://ecyojaidan.github.io" rel="nofollow noopener">ecyojaidan.github.io</a>” (don’t mess this up!  It needs to be exactly <em>your_username.github.io</em>, or else it won’t work).  Then all you have to do is commit and push your website code to your new repository!  And just like that, your website is now live :tada:</p>
<p>Please tell me if you have any remaining questions!</p>

**Comment by OLUBEST** [comment]
<p>Okay i appriciate that when that is possible you tell me </p>

**Comment by nanisadw3** [comment]
Here is the quickest step-by-step way to set up your personal GitHub Page for a resume or portfolio:
  
  1. **Create a new repository:**
     - Name it specifically: `<your-github-username>.github.io` (must match your username exactly).
     - Set visibility to **Public**.
  2. **Add your files:**
     - Create an `index.html` file in the root of the repository with your resume content.
  3. **Enable Pages:**
     - Go to **Settings** -> **Pages**.
     - Under **Build and deployment**, ensure **Source** is set to "Deploy from a branch", branch: `main`,
folder: `/(root)`.
  4. **Visit your site:**
     - Within 1–2 minutes, your website will be live at `https://<your-github-username>.github.io`.
  
  Whenever you push commits to `main`, your site updates automatically.
  
  If this guided you in the right direction, please consider marking it as the accepted answer!
